src/appleseed/foundation/meta/tests/test_statistics.cpp

```diff
@@ -185,6 +185,32 @@ TEST_SUITE(Foundation_Utility_Statistics)
 
         EXPECT_EQ("  existing value                17,042", stats.to_string());
     }
+
+    TEST_CASE(Merge_GivenExistingPercentStatistic_InsertsIt)
```

Comment:
The title is incorrect here: it should be `Merge_GivenNewPercentStatistic_InsertsIt`